doc/getting-started.md

```diff
@@ -23,17 +23,30 @@ The configuration must be passed to Ignition through the designated data source.
 
 ## Troubleshooting
 
+### Gathering Logs
+
 The single most useful piece of information needed when troubleshooting is the log from Ignition. Ignition runs in multiple stages so it's easiest to filter by the syslog identifier: `ignition`. When using systemd, this can be accomplished with the following command:
 
 ```
-journalctl --identifier=ignition
+journalctl --identifier=ignition --all
 ```
 
 In the event that this doesn't yield any results, running as root may help. There are circumstances where the journal isn't owned by the systemd-journal group or the current user is not a part of that group.
 
-In the vast majority of cases, it will be immediately obvious why Ignition failed. If it's not, inspect the config that Ignition wrote into the log. This shows how Ignition interpreted the supplied configuration. The user-provided config may have a misspelled section or maybe an incorrect hierarchy.
+### Validating the Configuration
+
+One common cause for Ignition failures is a malformed configuration (e.g. a misspelled section or incorrect hierarchy). Ignition will log errors, warnings, and other notes about the configuration that it parsed, so this can be used to debug issues with the configuration provided. As a convenience, CoreOS hosts an [online validator][validator] which can be used to quickly verify configurations.
+
+### Enabling Services
```

Comment:
To the casual reader, this hops from Ignition terms to systemd terms, whereas elsewhere you say "when using systemd". Maybe title this "Systemd Services" or "Enabling Systemd Services" or "Enabling Units"